var/spack/repos/builtin/packages/py-chemfiles/package.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyChemfiles(PythonPackage):
+    """Python interface to chemfiles"""
+
+    homepage = "http://chemfiles.org/chemfiles.py/latest/"
+    pypi = "chemfiles/chemfiles-0.10.3.tar.gz"
+
+    maintainers("RMeli")
+
+    version("0.10.3", sha256="4bbb8b116492a57dbf6ddb4c84aad0133cd782e0cc0e53e4b957f2d93e6806ea")
+
+    depends_on("chemfiles@0.10.3+shared", when="@0.10.3")
+
+    depends_on("python")
```

Comment:
This can be removed, it's already added by the base class

Reply:
```suggestion
```